aws/README.md

```diff
@@ -0,0 +1,99 @@
+# Overview
+
+## Directory structure
+
+This project contains source code and supporting files for a serverless application that you can deploy with the AWS Serverless Application Model (AWS SAM) command line interface (CLI). It includes the following files and folders:
+
+- `src` - Code for the application's Lambda function.
+- `events` - Invocation events that you can use to invoke the function.
+- `__tests__` - Unit tests for the application code.
+- `template.yaml` - A template that defines the application's AWS resources e.g. functions
+
+# Preparation
+
+- Install [Docker](https://www.docker.com/) and [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html). We need Docker running in the background for AWS SAM to work. We don't need to interact directly with it.
+  - **Notes**: At the moment it seems some new versions of Docker may not work well with AWS SAM CLI, the latest safe version was [**4.7.0**](https://docs.docker.com/desktop/release-notes/#docker-desktop-470). It's possible newer versions may not have this issue. Recommended to install the latest first, then if AWS SAM complains about no Docker found, revert to the older version.
+- Run `npm i` inside this directory
+- Create a `env.local.json` file based on the existing `env.template.json`. This file helps run the API server with appropriate parameters during development
+  - For example, `ES_URL` and `PG_URL` can be obtained directly from the corresponding Aiven services (OpenSearch & PostgreSQL)
```

Comment:
I think these are development instructions, so let's instruct people to create an OpenSearch and PostgreSQL service themselves, and use those. There's no reason to share or use the live services other than for production, the credentials have not been shared before (Ops obviously have access, we're using Aiven services)